src/plugins/default/DuplicateCheckerPlugin-spec.ts

```diff
@@ -1,64 +1,63 @@
-import { ContextData } from '../ContextData';
+import { ExceptionlessClient } from '../../ExceptionlessClient';
 import { EventPluginContext } from '../EventPluginContext';
 import { DuplicateCheckerPlugin } from './DuplicateCheckerPlugin';
-import { ErrorPlugin } from './ErrorPlugin';
-import { createFixture } from './EventPluginTestFixture';
 import { expect } from 'chai';
 
 describe('DuplicateCheckerPlugin', () => {
   let now: number = 0;
+  let client: ExceptionlessClient;
   let plugin: DuplicateCheckerPlugin;
 
   beforeEach(() => {
-    plugin = new DuplicateCheckerPlugin(() => now);
+    client = new ExceptionlessClient('LhhP1C9gijpSKCslHHCvwdSIz298twx271n1l6xw', 'http://localhost:50000');
+    plugin = new DuplicateCheckerPlugin(() => now, 50);
   });
 
   function run(exception: Error) {
-    let context: EventPluginContext;
-    let contextData: ContextData;
-    ({ context, contextData } = createFixture());
+    let errorParser = client.config.errorParser;
```

Comment:
Well for simple tests like this, I figured it would be better to just run the plugins that are required so it's faster and you are running only what is needed.